test/inductor/test_fp8.py

```diff
@@ -475,6 +475,9 @@ def linear(x_fp8, x_inverse_scale, w_t_fp8, w_inverse_scale, bias):
                 self.assertEqual(y_eager, y_compiled, rtol=1e-2, atol=0.05)
 
     @unittest.skipIf(not PLATFORM_SUPPORTS_FP8, f8_msg)
+    # TODO (eellison): fails with:
+    # "RuntimeError: mat2 must be col_major, got stride (64, 1)".
+    @unittest.expectedFailure
```

Comment:
why is this a todo for elias lol

Reply:
@zou3519 Because the test has been silently broken since he wrote it, and I'm frankly unsure how to fix it. :sweat_smile: I was hoping that he would know where to go if I left a todo.